packages/framework/src/Foundation/Concerns/ForwardsFilesystem.php

```diff
@@ -28,21 +28,11 @@ public function vendorPath(string $path = '', string $package = 'framework'): st
         return $this->filesystem->vendorPath($path, $package);
     }
 
-    public function mediaPath(string $path = ''): string
-    {
-        return $this->filesystem->mediaPath($path);
-    }
-
     public function sitePath(string $path = ''): string
     {
         return $this->filesystem->sitePath($path);
     }
 
-    public function siteMediaPath(string $path = ''): string
-    {
-        return $this->filesystem->siteMediaPath($path);
-    }
-
```

Comment:
This means they are lost from the HydeKernel which could be a problem to review before merge.

Reply:
Think we can revert the deprecated compat trait, as root deprecations should propagate after https://github.com/hydephp/develop/pull/1919

Reply:
Don't think this is an issue anymore.